services/order.py

```diff
@@ -0,0 +1,32 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> Order:
+    user = get_user_model().objects.get(username=username)
+    with transaction.atomic():
+        order = Order.objects.create(user=user,)

```

Comment:
You are using single quotes in your code. It's better to stick to one style of quotes in your code. Double quotes are preferable.